docs/index.html

```diff
@@ -213,17 +213,17 @@
       },
       {
         quadrant: 2,
-        ring: 0,
+        ring: 3,
         label: "Ember.js",
         active: false,
-        moved: 0
+        moved: -2
       },
       {
         quadrant: 2,
-        ring: 2,
+        ring: 0,
         label: "Lottie",
         active: false,
-        moved: 0
+        moved: 1
```

Comment:
isn't this a +2 move? or is only relevant if it's positive or negative?

Reply:
Just positive/negative - it is not visualised on the radar how far it is moved.